default/stringtables/en.txt

```diff
@@ -5681,7 +5684,7 @@ Each [[encyclopedia ENC_SPECIES]] has its own environmental preferences, which d
 
 [[PE_UNINHABITABLE]] < [[PE_HOSTILE]] < [[PE_POOR]] < [[PE_ADEQUATE]] < [[PE_GOOD]]
 
-The planet suitability report can be displayed by right-cliking on a planet in the system sidepanel. A green number following the suitability data indicates the max population value if the species colonizes the planet, whereas a red number indicates that the population will stay idle or will decrease, to finally perish if the species attempts to colonize the planet.
+The planet suitability report can be displayed by right-cliking on a planet in the system sidepanel. A green number following the suitability data indicates the maximum [[metertype METER_POPULATION]] the planet could achieve if colonized by this species, whereas a red number indicates that the Population will stay idle or will decrease, to finally perish if the species attempts to colonize the planet.
```

Comment:
that the colony will be unviable, and population will decrease over several turns until the the colonists have all died off and the colony is lost.

Reply:
Ok, I will modify it.

Reply:
...?